assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentToolUtils.java

```diff
@@ -679,6 +679,7 @@ else if ("remove".equals(addUpdateRemoveAssignment)) {
                 }
             }
         } catch (Exception e) {
+            log.error(e.getMessage());
```

Comment:
You should write your own message here, and then append e.toString(). e.toString gives us the class name and line number, so it is more helpful when debugging.